utils/bigclown/bigclown-mqtt2influxdb/files/init

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=1
+
+USE_PROCD=1
+
+PROG=/usr/bin/mqtt2influxdb
+CONF=/etc/bigclown-mqtt2influxdb.yml
+
+start_service() {
+	procd_open_instance
+	procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
```

Comment:
curios: where do you get these variables from ? 
`respawn_threshold`, `respawn_timeout` & `respawn_retry` ?

Reply:
Hmm, not sure. Copy-paste from procd documentation. I think that idea is that you can set those in uci but I am not retrieving them from uci so I am going to drop them.